sprint8/project/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Home cooking</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+  <div class="container">
+    <div class="container__colo1r">
+    <div class="container__color--yellow">Eggs</div>
+    <div class="container__color--red">Bacon</div>
+    <div class="container__color--pink">Donuts</div>
+    <div class="container__color--beige">Toast</div>
+    <div class="container__color--brown">Coffe</div>
```

Comment:
Mamy tu 5 divów. Ma byc jedna klasa powiedzmy że nazwiemy ja `superKlasa`  i ta klasa ma mieć style które będą się powtarzać dla każdego diva (np. szerokość, czcionka itd...). dodatkowo każdy div ma mieć drugą klasę i teraz ta klasa ma być modyfikatorem, czyli na koniec każdy z divów będzie miał 2 klasy. Jedną wspólną, drugą modyfikator. Te modyfikatory by wtedy były takie: `superKlasa--żółta`, `superKlasa--czerwona`. Oczywiście prosze nie wpisywac `superKlasa` tylko wymyślic swoje nazwy. Te co są teraz do wymiany. Klasy niech odzwierciadlają co jest w elementach.  w tym przypadku to jedzenie, produkty, zakupy :) coś w ten deseń oczywiście po angielsku :)